### ISSUE TITLE
parsePlugin.initialize is never run on ios

### ISSUE THREAD

**Issue by Tiagojdferreira** [avivais/phonegap-parse-plugin#53]
I have the parsePlugin working on Android, but on iOS it arrives at:

``` javascript
return parsePlugin.initialize('asd', 'qwe', (function() {
...
}
```

It goes to:

``` javascript
    initialize: function(appId, clientKey, successCallback, errorCallback) {
        cordova.exec(
            successCallback,
            errorCallback,
            'ParsePlugin',
            'initialize',
            [appId, clientKey]
        );
    },
```

But the original code inside initialize is never run,

**Comment by Tiagojdferreira** [comment]
This plugin still requires you to activate Push Notifications for your App Id and generate and add the SSL certificate to parse?

**Comment by devHasitha** [comment]
Did you find the reason for that. Coz I'm also have this problem.

**Comment by Pentiado** [comment]
THREAD WARNING: ['ParsePlugin'] took '32.885986' ms. Plugin should use a background thread.
This is the message I'm getting in xcode console when trying to run initialize and none of the callbacks is ever run

**Comment by MarcBT** [comment]
Hi @Tiagojdferreira, same question as @devHasitha ? 
Did you find a solution to this issue ? 
Thanks in advance !

**Comment by tiagojdf** [comment]
I am not sure anymore what was the problem at the time, and I am sorry I did not share the answer once I found it. 
Here are some possible problems:
Be sure to call the function right after the module is initialized with the .run($ionicPlatform) function and within the $ionicPlatform.ready function
Be sure your plugin is well registered (at some point I installed the local push plugin instead of this one, due to their similarity in name) and that it gets built. To debug, use xcode
Create the Apple Push SSL certificate, upload it to the Parse server.
